<launch>

    <node pkg="iss_manager" type="planning_manager_node.py" name="planning_manager_node" output="screen">
        <remap from="planning/lanelet2_planner/trajectory" to="planning/global_planner/trajectory"/>
        <remap from="planning/lattice_planner/trajectory" to="planning/local_planner/trajectory"/>
        <param name="local_planning_frequency" value="5"/>
    </node>

    <node pkg="iss_manager" type="control_manager_node.py" name="control_manager_node" output="screen">
        <param name="control_frequency" value="20"/>
        <remap from="control/wpt_tracker/control_command" to="control/control_command"/>
    </node>

    <include file="$(find carla_bridge)/launch/carla_bridge.launch"/>

</launch>
